### DISCUSSION TITLE
npm install not working

### DISCUSSION THREAD

**Discussion by chs000-00** [vercel/community#3199] [Help]
### Summary

Vercel does not install npm modules into node_modules.

### Example

https://github.com/Igbalabanov/NPM

### Steps to Reproduce

1. Deploy a project to vercel that uses npm or yarn
2. Node Modules folder not found

**Comment by amyegan** [comment]
Hi @Igbalabanov. Please walk me through your deployment steps. 🙏 I'm most interested in knowing... 

- Are you using the default [Vercel Git integration](https://vercel.com/docs/concepts/deployments/git/vercel-for-github) or [the CLI](https://vercel.com/docs/cli) to build and deploy?
- What should the [Build & Development settings](https://vercel.com/docs/concepts/deployments/configure-a-build#build-and-development-settings) be for the project?
- Where should I see an error related to `node_modules`? Will it be in the [build logs](https://vercel.com/docs/concepts/deployments/logs), somewhere on the site after deployment, or in the [runtime logs](https://vercel.com/docs/concepts/observability/runtime-logs)? What is the error message I should see?

**Comment by chs000-00** [comment]
Hello @amyegan, I am using the Vercel Git integration to deploy.
My Build & Development settings are as follows:

- Build Command: auto
- Output Directory: auto
- Install Command: `npm install`
- Development Command: auto

I do not see any error relating to node_modules, yet It returns a 404 error whenever I try to access it
